j2k/testData/fileOrElement/switch/caseWithDefaultblock.java

```diff
@@ -0,0 +1,21 @@
+//method
```

Comment:
ok, just to confirm, below file right?
 kotlin/j2k/tests/org/jetbrains/kotlin/j2k/JavaToKotlinConverterSingleFileTestGenerated.java